src/pages/SignUp/components/SignupForm.tsx

```diff
@@ -156,9 +167,19 @@ const SignUpForm: React.FC<SignUpFormProps> = ({
     }
   };
 
-  const handleShowPassword = () => {
-    setShowPassword(showPassword ? false : true);
-  };
+  
+
+  const passwordInput = (
+    <TextInput
+      isRequired
+      type={passwordHidden ? 'password' : 'text'}
+      id="pf-login-password-id"
+      name="pf-login-password-id"
+      validated={isValidPassword ? ValidatedOptions.default : ValidatedOptions.error}
+      value={passwordValue}
+      onChange={onChangePassword}
```

Comment:
I don't see an `onChangePassword` event handler. We probably need to  set state of the input right?

Reply:
Ohh, in LoginForm.tsx it was used directly so I got confused. I will have a look at it then.

Reply:
No worries. Good catch. Simply put,

The value of the `passwordValue` would be an `' '`.  It's being set to a default value.
Simply put, the fix is to handle the `onChangeEvent` on the `TextInput`. Set the value you receive to the `passwordState` and use that as your value.

Reply:
Okay, thanks for your guidance. I will give it a try